src/modules/Lan.rb

```diff
@@ -703,11 +683,9 @@ def Import(settings)
       else
         NetworkService.use_wicked
       end
-      if Builtins.haskey(settings, "ipv6")
-        @ipv6 = Ops.get_boolean(settings, "ipv6", true)
-      end
 
-      LanItems.modified = true
+      @ipv6 = settings["ipv6"] || true
```

Comment:
it is a bit confusing to have spread over whole file settings default. What about set it at beginning or maybe even better use merging? like `settings = DEFAULT_SETTINGS.merge(settings)`